opengrok-indexer/src/main/java/org/opengrok/indexer/util/Statistics.java

```diff
@@ -28,28 +28,47 @@
 import static org.opengrok.indexer.util.StringUtils.getReadableTime;
 
 public class Statistics {
-        
-  private final long startTime;  
 
-  public Statistics() {
+    private final long startTime;
+
+    public Statistics() {
       startTime = System.currentTimeMillis();    
   }
 
-  public void report(Logger log, String msg) {
-      long stopTime = System.currentTimeMillis();
-      String time_str = StringUtils.getReadableTime(stopTime - startTime);
-      log.log(Level.INFO, msg + " (took {0})", time_str);
-  }
+    /**
+     * log a message along with how much time it took since the constructor was called.
+     * @param logger logger instance
+     * @param logLevel log level
+     * @param msg message string
+     */
+    public void report(Logger logger, Level logLevel, String msg) {
+        long stopTime = System.currentTimeMillis();
+        String time_str = StringUtils.getReadableTime(stopTime - startTime);
```

Comment:
I know it was like this but since you are touching it, could you please rename to `timeStr` to follow Java standard?

Reply:
sure